src/main/java/com/aws/greengrass/deployment/activator/KernelUpdateActivator.java

```diff
@@ -70,9 +73,12 @@ public void activate(Map<String, Object> newConfig, Deployment deployment,
 
         DeploymentDocument deploymentDocument = deployment.getDeploymentDocumentObj();
         KernelLifecycle lifecycle = kernel.getContext().get(KernelLifecycle.class);
+
+        Set<String> servicesConfig = newConfig.get(SERVICES_NAMESPACE_TOPIC) == null ? Collections.emptySet() :
```

Comment:
Cal it `services` or `serviceNames` or something that doesn't give an impression that this has all service config like lifecycle, component config, etc.